### ISSUE TITLE
Learning Robust Deep Face Representation

### ISSUE THREAD

**Issue by naranjuelo** [AlfredXiangWu/face_verification_experiment#18]
Hi Alfred Xiang Wu:

I've read your paper "Learning Robust Deep Face Representation", where you proposed a CNN architecture with Max-Feature-Map (MFM) activation functions. I compared it with the prototxt you uploaded here and I'm a little confused about the differences. 

How do you apply your MFM layer? A convolutional layer, followed by a pooling layer, a slicing layer and an eltwise layer (prototxt) is the same as two consecutive convolutional layers, a MFM layer and a pooling layer (paper)?

Thank you very much!

**Comment by AlfredXiangWu** [comment]
For example, we assume the output of conv1 is 120*120*96. We can use slice layer to divide the conv1 into conv1_1(120*120*48) and conv1_2(120*120*48) and then we use eltwise layer to get the output(120*120*48) by computing the max value between two groups of  feature maps.

**Comment by naranjuelo** [comment]
Ok, I understand it now. So if I would like to train the model with ReLU layers instead of MFM, I would replace the layers convolution-pooling-slice-eltwise with convolution-pooling-ReLU, am I right?

**Comment by AlfredXiangWu** [comment]
The order should be convolution-ReLU-pooling in theory, but I think it doesn't influence the results whichever orders you choose.

**Comment by naranjuelo** [comment]
Ok! Yes, it should be the same. 

Regarding to the same paper I have another question. "The learning rate is set to 1e-3 initially and reduce to 5e-5 gradually". Did you use any specific learning rate policy (step down, exponential decay...)? Or was it a fixed learning rate (1e-3), manually decreased to 5e-5? Thanks!

**Comment by AlfredXiangWu** [comment]
The policy of learning rate is fixed and I manually changed it if the loss isn't decreased for a long time.

**Comment by naranjuelo** [comment]
Ok, understood! Thank you very much
